.travis.yml

```diff
@@ -1,12 +0,0 @@
-language: c
```

Comment:
no worries at all, and thanks for sharing your work! could you add the `.travis.yml` back to your fork when you get a chance? it would help us double check the pull req.